app/lib/features/space/widgets/links_card.dart

```diff
@@ -34,7 +34,7 @@ class LinksCard extends ConsumerWidget {
                   if (pins.isEmpty) {
                     return [
                       Text(
-                        L10n.of(context).thereAreNoItemInThisSpace('pins'),
+                        L10n.of(context).thereAreNoPinsInThisSpace,
```

Comment:
May I suggest to stick with simple keys rather than writing out entire sentences...?!?
```suggestion
                        L10n.of(context).noPinsInSpace,
```